### PR TITLE
Update gradle plugin version from 1.5.0 to 2.1.0

### PR DESCRIPTION
**Pull request by chthai64**
Shouldn't we update the gradle version to 2.1.0?

### CODE REVIEW

**Comment by oprisnik** [comment]
Thanks for the PR @chthai64. Unfortunately this breaks the build, see https://circleci.com/gh/facebook/fresco/128

It seems that simply updating the version number is not enough.

**Comment by chthai64** [comment]
I see. I will try to figure out why it breaks.

**Comment by facebook-github-bot-3** [comment]
@chthai64 updated the pull request.

**Comment by chthai64** [comment]
The problem of 26007ad is the project imagepipeline-test has incorrect package name "com.facebook.imagepipelinebase" which is ignored by older gradle version, so I corrected it. However, it failed the test on CircleCI due to 4Gb memory limit, but works on local machine.

**Comment by kirwan** [comment]
@facebook-github-bot import

**Comment by facebook-github-bot-2** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/819775191499618/int_phab) to review.

File: imagepipeline-test/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest
     xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.facebook.imagepipelinebase"
+    package="com.facebook.imagepipeline.testing"
```
Comment (massimocarli) [inline imagepipeline-test/src/main/AndroidManifest.xml]: please use `com.facebook.imagepipeline.test`

**Comment by chthai64** [comment]
I just changed it to ```com.facebook.imagepipeline.test```

**Comment by facebook-github-bot-5** [comment]
@chthai64 updated the pull request.

**Comment by kirwan** [comment]
@chthai64 Could you please rebase again?

I've made a change which I hope will resolve Circle CI's memory problem but it would be great to confirm that before merging the PR.

Thanks so much for your help with this.

**Comment by chthai64** [comment]
@kirwan I just merged the last two commits into one.

**Comment by facebook-github-bot-8** [comment]
@chthai64 updated the pull request.

**Comment by kirwan** [comment]
Thanks. Circle is happy now so we can get this merged.

**Pull request closed without merging**